### PR TITLE
Add the body bag

### PR DESCRIPTION
**Pull request by Autisem**
# What this PR does
Adds body bags. You may purchase them from the janie vendor

TODO:

- [x] Limit to 1 corpse(literly just a deployable crate right now)
- [x] Port sprites

# Why it should be added to the game
The cadaver bus was on your wish list

### CODE REVIEW

**Comment by Autisem** [comment]
On a few notes, seems that structures check if the floor is constructable, maby a argument to bypass this would be nice as something like body bag's should be deployable station side

**Comment by Autisem** [comment]
Default TG sprites for body bags use overlays to get the open and close effect instead of premade sprites. Place holder will be coffin sprite for now

**Comment by Autisem** [comment]
it doesnt accept new PR'ed meme didnt even edit map smh

**Comment by BurgerLUA** [comment]
Is this a WIP PR?

**Comment by Autisem** [comment]
> 
> 
> Is this a WIP PR?

Its almost done, I havent tested yet if the bodybag fits one corpse but I added the size var to it, sadly cant test it now though since its quite late

File: code/_core/obj/item/fluff/bikehorn.dm [hunk 1]
```diff
@@ -18,7 +18,9 @@ obj/item/bikehorn/click_self(var/mob/caller)
 	INTERACT_DELAY(20)
 
 	var/turf/T = get_turf(src)
+  
 	play_sound('sound/items/bikehorn.ogg',T)
+
```
Comment (BurgerLUA) [inline code/_core/obj/item/fluff/bikehorn.dm:23]: Try to avoid empty commits to files in the future.

**Review by BurgerLUA — APPROVED** [review]

File: code/_core/obj/item/fluff/bikehorn.dm (hunk 1 above)
Comment (Autisem) [reply code/_core/obj/item/fluff/bikehorn.dm:23]: was a merge conlfict, but ill try

**Pull request merged**